lib/ansible/modules/cloud/docker/docker_network.py

```diff
@@ -378,12 +398,14 @@ def main():
         appends=dict(type='bool', default=False, aliases=['incremental']),
         ipam_driver=dict(type='str', default=None),
         ipam_options=dict(type='dict', default={}),
+        internal=dict(type='bool', default=None),
         debug=dict(type='bool', default=False)
     )
 
     client = AnsibleDockerClient(
         argument_spec=argument_spec,
-        supports_check_mode=True
+        supports_check_mode=True,
+        min_docker_api_version='1.9.0'
```

Comment:
Change this to `min_docker_version`, as this is about docker-py and not the docker API

Reply:
Hah, whoops. I suppose the naming scheme for those makes some sort of sense. I'd prefer min_docker_py_version to be explicit.

Reply:
It's the usual problem of `docker-py` being called `docker` as well... :)